prog/postgres/postgres_resource_nexus.rb

```diff
@@ -120,7 +120,11 @@ def before_run
 
   label def wait_server
     nap 5 if server.strand.label != "wait"
-    hop_create_billing_record
+    if postgres_resource.active_billing_records.empty?
+      hop_create_billing_record
```

Comment:
Is it best to branch on this, or something else that is more tightly coupled to the initial provisioning, e.g. a semaphore that is initialized as incremented? It seems like binding a form of "rectification" to entering `wait_server` (which can happen for multiple reasons it seems) could have an excessive number of unintended consequences.

Reply:
PostgresServer has a `initial_provisioning` semaphore, we may add same one to PostgresResource too.

Reply:
Initially, I considered not entering the `wait_server` label. However, `refresh_certificates` increases the `refresh_certificates` semaphores of the servers. The servers transition to the `refresh_certificates` and `configure` labels. Should we wait for them to reach the `wait` state?

Reply:
I talked with @byucesoy offline. Certificate renewal is an online operation, it doesn't cause degradation. There's no need to wait for the server. We can hop to the 'wait' label after refreshing the certificate.